main/mafia.hs

```diff
@@ -344,7 +355,15 @@ mafiaBuild p w flags args = do
         EnableWarnings ->
           ["--ghc-options=-Werror"]
 
-  liftCabal . cabal_ "build" $ ["-j"] <> wargs <> args
+    dumpargs =
+      case dump of
+        DisableCoreDump ->
+          []
+        EnableCoreDump ->
+          ["--ghc-options=-ddump-prep -ddump-to-file -dppr-case-as-let -dsuppress-all"]
```

Comment:
You can mention `--ghc-options` multiple times, so I think I would prefer this to be:
```hs
[ "--ghc-options=-ddump-prep"
, "--ghc-options=-ddump-to-file"
, "--ghc-options=-dppr-case-as-let"
-- etc --
]
```